cmd/kubeadm/app/constants/constants.go

```diff
@@ -32,48 +32,82 @@ import (
+	// APIServerCertCommonName defines API's server certificate common name
+	APIServerCertCommonName = "kube-apiserver" //used as subject.commonname attribute (CN)
+
+	// APIServerKubeletClientCertAndKeyBaseName defines kubelet client certificate and key base name
 	APIServerKubeletClientCertAndKeyBaseName = "apiserver-kubelet-client"
-	APIServerKubeletClientCertName           = "apiserver-kubelet-client.crt"
-	APIServerKubeletClientKeyName            = "apiserver-kubelet-client.key"
-	APIServerKubeletClientCertCommonName     = "kube-apiserver-kubelet-client" //used as subject.commonname attribute (CN)
-
-	ServiceAccountKeyBaseName    = "sa"
-	ServiceAccountPublicKeyName  = "sa.pub"
+	// APIServerKubeletClientCertName defines kubelet client certificate name
+	APIServerKubeletClientCertName = "apiserver-kubelet-client.crt"
+	// APIServerKubeletClientKeyName defines kubelet client key name
+	APIServerKubeletClientKeyName = "apiserver-kubelet-client.key"
+	// APIServerKubeletClientCertCommonName defines kubelet client certificate common name
+	APIServerKubeletClientCertCommonName = "kube-apiserver-kubelet-client" //used as subject.commonname attribute (CN)
+
+	// ServiceAccountKeyBaseName defines SA key base name
+	ServiceAccountKeyBaseName = "sa"
+	// ServiceAccountPublicKeyName defines SA public key base name
+	ServiceAccountPublicKeyName = "sa.pub"
+	// ServiceAccountPrivateKeyName defines SA private key base name
 	ServiceAccountPrivateKeyName = "sa.key"
 
+	// FrontProxyCACertAndKeyBaseName defines front proxy CA certificate and key base name
 	FrontProxyCACertAndKeyBaseName = "front-proxy-ca"
-	FrontProxyCACertName           = "front-proxy-ca.crt"
-	FrontProxyCAKeyName            = "front-proxy-ca.key"
+	// FrontProxyCACertName defines front proxy CA certificate name
+	FrontProxyCACertName = "front-proxy-ca.crt"
+	// FrontProxyCAKeyName defaines front proxy CA key name
+	FrontProxyCAKeyName = "front-proxy-ca.key"
 
+	// FrontProxyClientCertAndKeyBaseName defines front proxy certificate and key base name
 	FrontProxyClientCertAndKeyBaseName = "front-proxy-client"
-	FrontProxyClientCertName           = "front-proxy-client.crt"
-	FrontProxyClientKeyName            = "front-proxy-client.key"
-	FrontProxyClientCertCommonName     = "front-proxy-client" //used as subject.commonname attribute (CN)
-
-	AdminKubeConfigFileName             = "admin.conf"
-	KubeletBootstrapKubeConfigFileName  = "bootstrap-kubelet.conf"
-	KubeletKubeConfigFileName           = "kubelet.conf"
+	// FrontProxyClientCertName defines front proxy certificate name
+	FrontProxyClientCertName = "front-proxy-client.crt"
+	// FrontProxyClientKeyName defines front proxy key name
+	FrontProxyClientKeyName = "front-proxy-client.key"
+	// FrontProxyClientCertCommonName defines front proxy certificate common name
+	FrontProxyClientCertCommonName = "front-proxy-client" //used as subject.commonname attribute (CN)
+
+	// AdminKubeConfigFileName defines kuberenetes config file name
```

Comment:
defines the file name for the KubeConfig aimed to be used by the superuser/admin of the cluster